### ISSUE TITLE
Unable to set up EKS cluster using Calico doc

### ISSUE THREAD

**Issue by lwr20** [projectcalico/vpp-dataplane#140]
**Environment**
- Calico/VPP version: 0.13.0
- Kubernetes version: 1.17
- Deployment type: EKS
- Network configuration: Applied https://raw.githubusercontent.com/projectcalico/vpp-dataplane/v0.13.0-calicov3.17.1/yaml/generated/calico-vpp-eks.yaml

**Issue description**
I'm trying to install using https://raw.githubusercontent.com/projectcalico/vpp-dataplane/v0.13.0-calicov3.17.1/yaml/generated/calico-vpp-eks.yaml

I'm following the instructions in https://deploy-preview-4383--calico-master.netlify.app/getting-started/kubernetes/vpp/eks

My calico-node pods are stuck pending - describe says:
```
  Warning  FailedScheduling  9s (x35 over 50m)  default-scheduler  0/3 nodes are available: 3 Insufficient hugepages-2Mi.
```
Cluster was installed using `eksctl` 0.30.0 and (by default) runs nodes with AmazonLinux2 and k8s 1.17

The doc didn't mention needing to configure the nodes with HugePages.  

I think either the doc needs to have instructions to enable HugePages or the manifest needs to be updated to not need them.

**To Reproduce**
- Follow https://deploy-preview-4383--calico-master.netlify.app/getting-started/kubernetes/vpp/eks

**Expected behavior**
Expected cluster to come up with all pods Ready.

**Additional context**
None.

**Comment by sknat** [comment]
Hi @lwr20 
I tried to reproduce this one but things seem to be working well
````bash
eksctl create cluster --name=some-test-cluster --nodes=3 --version=1.17
kubectl -n kube-system delete daemonset aws-node
curl -o calico-vpp-eks.yaml https://raw.githubusercontent.com/projectcalico/vpp-dataplane/v0.13.0-calicov3.17.1/yaml/generated/calico-vpp-eks.yaml
kubectl apply -f calico-vpp-eks.yaml
kubectl -n kube-system get pods
  NAME                                       READY   STATUS              RESTARTS   AGE
  calico-kube-controllers-6d4bfc7c57-wsf22   0/1     ContainerCreating   0          3h20m
  calico-vpp-node-lp8pw                      2/2     Running             0          3h20m
  calico-vpp-node-mzpwb                      2/2     Running             0          3h20m
  calico-vpp-node-xlfvk                      2/2     Running             0          3h20m
  coredns-5fdf64ff8-8k2kr                    1/1     Running             0          3h47m
  coredns-5fdf64ff8-dgkd9                    1/1     Running             0          3h47m
  kube-proxy-7hkrf                           1/1     Running             0          3h38m
  kube-proxy-grs6r                           1/1     Running             0          3h38m
  kube-proxy-qdsw6                           1/1     Running             0          3h38m
````
Nodes ended up with the AMI `amazon-eks-node-1.17-v20210322` a.k.a `ami-00db62ace442b9070`

**Comment by sknat** [comment]
#144 should improve the behavior with dpdk & hugepages enabled by adding a DaemonSet [0] the builds & inserts the proper igb_uio kernel module at startup. Not ideal for production, but should make testing a lot easier
[0] https://github.com/projectcalico/vpp-dataplane/pull/144/commits/276c46ed777228b61ddf47c3e268ec8e53ed2f91#diff-33412ab5d0f3f6677d100d7f9446ae4c72e63acbb8ee412f1ad6ab30cb127ddcR63

**Comment by sknat** [comment]
I'll redo the test from a blank cluster w/ dpdk to see if it works next weekm & keep you posted

**Comment by AloysAugustin** [comment]
This should be fixed in the 0.14 release, please reopen if it is not the case.
